code/__DEFINES/radio.dm

```diff
@@ -64,6 +68,7 @@
 #define FREQ_MEDICAL 1355  //! Medical comms frequency, soft blue
 #define FREQ_ENGINEERING 1357  //! Engineering comms frequency, orange
 #define FREQ_SECURITY 1359  //! Security comms frequency, red
+#define FREQ_INCURSION 1361  //!  Private channel for the incursion team
```

Comment:
This should be lower on the non accessible radio channels rather than in the middle of all the station channels